geo.txt

```diff
@@ -56,3 +56,1703 @@ Phys 91(11):957-962
+
+H H 0.92(1)
+
+NAME = BFH
+# Point Group: Cs
+# Shape: Bent
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Sunahori FX, Clouthier DJ (2009) The electronic spectrum of the fluoroborane free radical. II. Analysis of laserinduced fluorescence and single vibronic level emission spectra. J Chem Phys 130(16):164310/1-164310/10
+doi: 10.1063/1.3122031
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+B H   1.214(2)
+B F   1.3034(5)
+F B H 120.7(1)
+
+NAME = BFH2O
+# Point Group: Cs (syn)
+# Shape: Planar
+# Bond lengths: re^se     (angstroms)
+# Bond angles:  thetae^se (degrees)
+# Source Paper: Vogt N, Demaison J, Vogt J, Rudolph HD (2014) Why it is sometimes difficult to determine the accurate
+position of a hydrogen atom by the semiexperimental method: structure of molecules containing the OH or the
+CH3 group. J Comput Chem 35(32):2333-2342
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+B H   1.1900(1)
+B F   1.3200(3)
+B O   1.3464(3)
+O H   0.9584(3)
+H B F 119.38(10)
+F B O 117.203(5)
+B O H 122.65(3)
+
+NAME = BF2
+# Point Group: C2v
+# Shape: Bent
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Yang J, Ellis B, Clouthier DJ (2011) The complex spectrum of a “simple” free radical: The A-X band system of
+the jet-cooled boron difluoride free radical. J Chem Phys 135(9):094305/1-094305/9
+doi:10.1063/1.3624528
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+B F   1.3102(9)
+F B F 119.7(6)
+
+NAME = BF2HO
+# Point Group: Cs
+# Shape: Planar
+# Bond lengths: re(se)     (angstroms)
+# Bond angles:  thetae(se) (degrees)
+# Source Paper: Vogt N, Demaison J, Rudolph HD, Perrin A (2015) Interplay of experiment and theory: high resolution infrared
+spectrum and accurate equilibrium structure of BF2OH. Phys Chem Chem Phys 17(45):30440-30449
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+B F(1)   1.3239(2)
+B F(2)   1.3139(2)
+B O      1.3426(3)
+O H      0.9581(6)
+O B F(1) 122.31(2)
+B O H    113.09(4)
+O B F(2) 119.47(2)
+F B F    118.22(3)
+
+NAME = BH2
+# Point Group: C2v
+# Shape: Bent
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Sunahori FX, Gharaibeh M, Clouthier DJ, Tarroni R (2015) BH2 revisited: New, extensive measurements of
+laser-induced fluorescence transitions and ab initio calculations of near-spectroscopic accuracy. J Chem Phys
+142(17):174302/1-174302/13
+[http://dx.doi.org/10.1063/1.4919094]
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+B H   1.197(2)
+H B H 129.6(2)
+
+NAME = BH6N
+# Point Group: C3v
+# Shape: Trigonal pyramidal --- Trigonal pyramidal
+# Bond lengths: rm^(1)     (angstroms)
+# Bond angles:  thetam^(1) (degrees)
+# Source Paper: Demaison J, Lievin J, Császár AG, Gutle C (2008) Equilibrium structure and torsional barrier of BH3NH3. J
+Phys Chem A 112(19):4477-4482
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+B...N   1.6576(16)
```

Comment:
Here we have the same issue as in Ag---NH3. I'm not sure how the computer will know how to make the z-matrix based on the data given here alone (without the diagram for guidance).